src/ui/MessageBody/index.tsx

```diff
@@ -0,0 +1,131 @@
+import React, {ReactElement} from 'react';
+import './index.scss';
+import {
+  CoreMessageType,
+  getUIKitMessageType, getUIKitMessageTypes, isMultipleFilesMessage,
+  isOGMessage,
+  isTextMessage, isThumbnailMessage, isVoiceMessage,
+} from '../../utils';
+import {FileMessage, MultipleFilesMessage, UserMessage} from '@sendbird/chat/message';
+import OGMessageItemBody from '../OGMessageItemBody';
+import TextMessageItemBody from '../TextMessageItemBody';
+import FileMessageItemBody from '../FileMessageItemBody';
+import MultipleFilesMessageItemBody from '../MultipleFilesMessageItemBody';
+import VoiceMessageItemBody from '../VoiceMessageItemBody';
+import ThumbnailMessageItemBody from '../ThumbnailMessageItemBody';
+import UnknownMessageItemBody from '../UnknownMessageItemBody';
+import {useThreadMessageKindKeySelector} from '../../modules/Channel/context/hooks/useThreadMessageKindKeySelector';
+import {useStatefulFileInfoList} from '../../modules/Channel/context/hooks/useStatefulFileInfoList';
+import {SendBirdStateConfig} from '../../lib/types';
+import {Nullable} from '../../types';
+import {GroupChannel} from '@sendbird/chat/groupChannel';
+
+const MESSAGE_ITEM_BODY_CLASSNAME = 'sendbird-message-content__middle__message-item-body';
```

Comment:
Since the context between CSS and JS/TS is not shared, there may be room for typos, so managing it as constants seems like a very good idea.